staging/src/k8s.io/client-go/tools/clientcmd/client_config.go

```diff
@@ -131,6 +131,7 @@ func (config *DirectClientConfig) ClientConfig() (*restclient.Config, error) {
 
 	clientConfig := &restclient.Config{}
 	clientConfig.Host = configClusterInfo.Server
+	clientConfig.DisableCompression = true
```

Comment:
this seems odd... this is setting a non-zero value for a field in the `restclient.Config` that does not come from the provided or overridden `config` inputs... that is unique in this function, and I don't think we want it here